### PR TITLE
Feature/12 server creation

### PR DESCRIPTION
**Pull request by Djadanis**

### CODE REVIEW

**Review by tbellicha — APPROVED** [review]

**Review by CedricCORGE — APPROVED** [review]

File: dependencies/Makefile [hunk 1]
```diff
@@ -0,0 +1,210 @@
+# CMAKE generated file: DO NOT EDIT!
```
Comment (JohanCDev) [inline dependencies/Makefile:1]: Useless file

File: dependencies/cmake_install.cmake [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Install script for directory: /workspaces/R-Type/dependencies
```
Comment (JohanCDev) [inline dependencies/cmake_install.cmake:1]: Useless file

File: src/Common/Message/Message.h [hunk 3]
```diff
@@ -0,0 +1,58 @@
+/**
+ * @file Message.h
+ * @author your name (you@domain.com)
```
Comment (JohanCDev) [inline src/Common/Message/Message.h:3]: Missing information

File: src/Common/Message/MessageHeader.h [hunk 4]
```diff
@@ -0,0 +1,17 @@
+/**
+ * @file MessageHeader.h
+ * @author your name (you@domain.com)
```
Comment (JohanCDev) [inline src/Common/Message/MessageHeader.h:3]: Missing information

File: src/Common/Message/MessageType.h [hunk 5]
```diff
@@ -0,0 +1,14 @@
+/**
+ * @file MessageType.h
+ * @author your name (you@domain.com)
```
Comment (JohanCDev) [inline src/Common/Message/MessageType.h:3]: Missing information

File: src/Common/locked_queue.h [hunk 6]
```diff
@@ -0,0 +1,44 @@
+/**
+ * @file locked_queue.h
+ * @author your name (you@domain.com)
```
Comment (JohanCDev) [inline src/Common/locked_queue.h:3]: Missing information

File: src/ECS/ResourcesManager.cpp [hunk 7]
```diff
@@ -0,0 +1,67 @@
+/**
+ * @file RegisterManager.cpp
+ * @author your name (you@domain.com)
```
Comment (JohanCDev) [inline src/ECS/ResourcesManager.cpp:3]: Missing information

File: src/client/client.cpp [hunk 8]
```diff
@@ -0,0 +1,75 @@
+/**
+ * @file client.cpp
+ * @author your name (you@domain.com)
```
Comment (JohanCDev) [inline src/client/client.cpp:3]: Missing information

File: src/client/client.hpp [hunk 9]
```diff
@@ -0,0 +1,49 @@
+/**
+ * @file client.hpp
+ * @author your name (you@domain.com)
```
Comment (JohanCDev) [inline src/client/client.hpp:3]: Missing information

File: src/server/main2.cpp [hunk 10]
```diff
@@ -0,0 +1,38 @@
+/**
```
Comment (JohanCDev) [inline src/server/main2.cpp:1]: Weird name for a main file

**Review by JohanCDev — CHANGES_REQUESTED** [review]

**Comment by Djadanis** [comment]
Displaced this PR

**Pull request closed without merging**